CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 -   Removed unused classes `DecoratorUtilities` and `LabelUtilities` from Core 
     Visual Schema
 
-## Changes in January 2022
+## Changes in February 2022
```

Comment:
You'll need to just remove this as there is already Changes in February header above this one

Reply:
Oops, sorry! Removed it now.